vlib/v/checker/tests/globals/cast_expr_T_type_err.vv

```diff
@@ -0,0 +1,22 @@
+module main
+
+__global (
+	fo = A(0)
+	fo1 = &A(0)
+)
+
+fn main() {
+
+}
+
+module main
```

Comment:
Why is the code duplicated? 🤔

Reply:
Good catch. I will fix it when I fix the tests.

Reply:
Done